### PR TITLE
Swift: package test SDK

### PR DESCRIPTION
**Pull request by AlexDenisov**
This PR adds a test SDK used for basic testing of the Swift extractor.

### CODE REVIEW

File: swift/BUILD.bazel [hunk 1]
```diff
@@ -39,9 +39,27 @@ pkg_files(
...
+    actual = select({
+        "@bazel_tools//src/conditions:%s" % arch: "@swift_prebuilt_%s//:swift-test-sdk" % arch
+        for arch in ("linux", "darwin_x86_64", "darwin_arm64")
+    }),
+)
+
+pkg_files(
+    name = "swift-test-sdk-arch",
+    srcs = [":swift-test-sdk"],
+    prefix = "qltest/" + codeql_platform,
```
Comment (criemen) [inline swift/BUILD.bazel:53]: is this a copy-paste error? (namely, the `qltest` part)

**Review by criemen — COMMENTED** [review]
One question, otherwise LGTM.

File: swift/BUILD.bazel (hunk 1 above)
Comment (AlexDenisov) [reply swift/BUILD.bazel:53]: That's on purpose. This idea is inherited from C++ support, though I cannot see any other usage of `qltest` in other language packs.
@criemen do you have a better place in mind? I'm happy to change/move this part if needed.

Comment (criemen) [reply swift/BUILD.bazel:53]: I misunderstood what the PR is doing. I cleared that up now, this looks good to me 👍

**Review by criemen — APPROVED** [review]

**Pull request merged**